libc/cmake/modules/LLVMLibCCompileOptionRules.cmake

```diff
@@ -152,6 +152,7 @@ function(_get_common_compile_options output_var flags)
     endif()
     list(APPEND compile_options "-Wconversion")
     list(APPEND compile_options "-Wno-sign-conversion")
+    list(APPEND compile_options "-Wno-c99-extensions")
```

Comment:
silencing this warning since `_Complex` is part of C99.

Reply:
Can you add this comment to the cmake file?

Reply:
sure.